holidays.py

```diff
@@ -2194,6 +2196,68 @@ class NL(Netherlands):
     pass
 
 
+class Egypt(HolidayBase):
+    SUNDAY = 1
+
+    def __init__(self, **kwargs):
+        # http://www.iamsterdam.com/en/plan-your-trip/practical-info/public-holidays
+        self.country = "NL"
+        HolidayBase.__init__(self, **kwargs)
+
+    def _populate(self, year):
+
+        # Christmas
+        self[date(year, 1, 7)] = "عيد الميلاد المجيد"
```

Comment:
would you consider writing the holiday names in ASCII English instead of Unicode Arabic?